EventPlanning.BL/EventManager.cs

```diff
@@ -39,6 +39,46 @@ public List<Event> GetUserEvents(int userId)
             return repository.GetUserEvents(userId)?.ToList();
         }
 
+        /// <summary>
+        /// Returns partial event data
+        /// </summary>
+        /// <param name="eventId"></param>
+        /// <returns></returns>
+        public PartialEvent GetPartialEvent(int eventId)
+        {
+            var ev = repository.GetEvent(eventId);
+            if (ev != null)
+            {
+                var partialEvent = new PartialEvent()
+                {
+                    Title = ev.Title,
+                    Creator = GetUserName(ev.UserId),
+                    Participants = GetParticipants(ev.Id),
+                    MaxParticipants = GetMaxParticipants(ev.Id),
+                };
+
+                return partialEvent;
+            }
+
+            return null;
```

Comment:
If method has all of it's logic (most of the code) in an `if` statement, we should opt to revert it and return fast